### ISSUE TITLE
ngx-bootstrap - Clicking on attached element. 

### ISSUE THREAD

**Issue by suparnavg** [isaacplmann/ngx-tour#118]
Facing the same issue #84 (https://github.com/isaacplmann/ngx-tour/issues/84), only it is with ngx-bootstrap.

Using "ngx-bootstrap": "2.0.0-rc.0" and
"ngx-tour-ngx-bootstrap": "2.0.0",

On clicking any attached element in the middle of the tour, the popover appears next to it (where it should appear if the tour was ongoing) - but the tour has not actually "started" - so clicking on next, end etc. has no effect at all.

**Comment by suparnavg** [comment]
For now, I've hacked a way around this by setting the global value ngx-bootstrap's PopoverConfig "trigger" config to "".

Add the snippet below to the app.module.ts

```
// Set global config for the PopoverConfig format
export function getPopoverConfig(): PopoverConfig {
  return Object.assign(new PopoverConfig(), {
    triggers: ""
  });
```
and 
`providers: [..., { provide: PopoverConfig, useFactory: getPopoverConfig }, ...]`

This seems to prevent the default "click" trigger event as defined in [PopoverConfig](https://valor-software.com/ngx-bootstrap/#/popover#popover-config)

**Comment by suparnavg** [comment]
I have a release coming up and hope to be able to get around to this in the next couple of weeks!

**Comment by AsmaaTahoon** [comment]
any update about this issue?

**Comment by isaacplmann** [comment]
Anyone is welcome to make a PR.  Let me know if you need help getting started.

**Comment by benj0c** [comment]
Can you help me with setup project in local env?

**Comment by isaacplmann** [comment]
Yep.

1. Fork the repo.
2. Clone the repo to you local machine
3. `npm run build:lib:all`
4. `npm start`

Let me know if you need help with any of those steps.

**Comment by benj0c** [comment]
Here is PR for this issue #147

**Comment by bartromgens** [comment]
I can confirm that this is solved in release 4.0.1. Thanks!
